netneurotools/plotting.py

```diff
@@ -228,6 +228,7 @@ def plot_conte69(data, lhlabel, rhlabel, surf='midthickness',
         Scene object containing plot
     """
 
+
```

Comment:
Can we get rid of this extra blank line? I don't think the linter is gonna like it.